index.html

```diff
@@ -69,30 +70,200 @@ <h2 class="intro_text">I am a Software developer. <br /></h2>
+    <section class="second_section_container1" id="mobile_portfolio">
+      <div class="second_section_div">
+      <div class="recent_work">
+      <div><h3 class="recent_work">My Recent Works</h3>
+      <hr class="line_divider" />
+      </div>
       <div class="image_holder">
-        <!-- multiple stories image comes here -->
+         <img class="work-main-image" src="images/Img Placeholder.png.png" />
       </div>
       <div><h3 class="stories">Multi-Post Stories</h3></div>
-      <div>
+      <div class="work_description_">
         <p class="work_description">
           A daily selection of privately personalized reads; no accounts or
           sign-ups required. has been the industry's standard dummy text ever
           since the 1500s, when an unknown printer took a standard dummy text.
         </p>
       </div>
-      <div class="languages">
-        <li class="boxes">Html</li>
-        <li class="boxes">Css</li>
-        <li class="boxes">Ruby</li>
-        <li class="boxes">Javascript</li>
+      <ul class="languages">
+        <li>Html</li>
+        <li>Css</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      
+       <span><button class="see_button">See Project</button></span> 
+      </div>
+      </div>
+    </section>
+    <section class="second_section_container2" id="mobile_portfolio2">
+      <div class="bg_image">
+            <img class="bg_image_bg" src="images/bg_image.png">
+          </div>
+      <div class="work_content">
+            <h3 class="work_title_art">
+              Profesional Art Printing Data
+            </h3>
+            <p class="work_description2">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div class="languages2">
+            <ul class="language_group2">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <div class="second_section_footer">
+            <a class="footer_fullwidth"> See Project </a>
+          </div>
+    </section>
+    <section class="third_section_container">
+      <div class="about_us_div">
+          <h2 class="about_title">About me</h2>
+          <p class="about_content">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+           <div class="resume_button">
+       <a class="button_resume"> Get My Resume </a>
+      </div>
+        </div>
+        <div class="line_divider_container">
+          <hr class="line_divider2" />
+        </div>
+         <div class="image_frame_holder">
+         <img class="frame_image" src="images/Frame.png.png" />
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. Please remove this tag and add this image as a background using CSS. Please remember that decorations are always background images, not `img` tags. :nerd_face: :nerd_face: 

# Your design
![image](https://user-images.githubusercontent.com/98363075/193412052-56d94f68-fbd7-4cde-b169-7d33fcceab3d.png)

# Figma design:
![image](https://user-images.githubusercontent.com/98363075/193412070-934cd33d-421e-47c2-a7f1-efa0f3562d02.png)

Reply:
@Santiago220991 Thanks for your review! I dont know where that distortion came from because i effected that change before. i have just added it as a backgroung image using css.